hw/top_englishbreakfast/util/prepare_sw.py

```diff
@@ -29,7 +29,7 @@
 BAZEL_BINARIES = [
     '//sw/device/lib/testing/test_rom',
     '//sw/device/sca:aes_serial',
-    '//sw/device/tests:aes_smoketest_prog',
+    '//sw/device/tests:aes_smoketest_sim_verilator',
```

Comment:
Hmm I think the problem is you want a [filegroup (or equivalent)](https://cs.opensource.google/opentitan/opentitan/+/master:rules/opentitan.bzl;drc=338ac2e474794a42d5fcb17b9bf0fafadb3e8f94;l=1218)for the flash binaries across all the devices. So maybe this should just be:
```python
'//sw/device/tests:aes_smoketest_sim_verilator',
'//sw/device/tests:aes_smoketest_sim_dv',
'//sw/device/tests:aes_smoketest_fpga_cw305',
'//sw/device/tests:aes_smoketest_fpga_cw310',
'//sw/device/tests:aes_smoketest_fpga_cw340',
```
?

Reply:
Fixed.